cfy_manager/components/restservice/db.py

```diff
@@ -205,18 +203,21 @@ def populate_db(configs, additional_config_files=None):
     logger.notice('Populating DB and creating AMQP resources...')
     args_dict = create_populate_db_args_dict()
     run_script('create_tables_and_add_defaults.py', args_dict, configs)
-    if (
-        config[MANAGER][SECURITY][ADMIN_USERNAME] and
-        config[MANAGER][SECURITY][ADMIN_PASSWORD]
+    args = ['manager_rest.configure_manager']
+    for file_name in (
+        constants.DEFAULT_CONFIG_PATH,
+        constants.USER_CONFIG_PATH,
```

Comment:
do we need this? I thought this is already in `config['config_files']` (which is appended 5 lines below)

Reply:
no, it's not, only these three files: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/bd4167431eaef42f46afc741c8850d2165c4b798/cfy_manager/components/restservice/restservice.py#L240-L244

Reply:
I mean this: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/bd4167431eaef42f46afc741c8850d2165c4b798/cfy_manager/config.py#L104
Passed in here: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/bd4167431eaef42f46afc741c8850d2165c4b798/cfy_manager/components/restservice/db.py#L215

`configs` (that dict) aren't even passed to configure_manager.py

Reply:
Indeed, `/etc/cloudify/config.yaml` is passed twice, so no need for `constants.USER_CONFIG_PATH` I suppose.  I am quite certain that lack of `constants.DEFAULT_CONFIG_PATH` was causing problems though.  Let's find out.